packages/terra-scroll/src/Scroll.jsx

```diff
@@ -21,15 +20,11 @@ const Scroll = ({
   children,
   refCallback,
   ...customProps
-}) => {
-  const theme = React.useContext(ThemeContext);
-
-  return (
-    <div {...customProps} className={cx(['scroll', customProps.className, theme.className])} ref={refCallback}>
-      {children}
-    </div>
-  );
-};
+}) => (
+  <div {...customProps} className={cx(['scroll', customProps.className])} ref={refCallback}>
```

Comment:
why are we removing themes support from terra-scroll ?? This PR is supposed to updated only content-container I believe !!